tutorials/7-cosmjs/5-create-custom.md

```diff
@@ -31,120 +31,38 @@ This section assumes that you have a working Cosmos blockchain with its own modu
 
 ## Compiling the Protobuf objects and clients
 
-You can choose which library you use to compile your Protobuf objects into TypeScript or JavaScript. Reproducing [what Stargate](https://github.com/cosmos/cosmjs/blob/main/packages/stargate/CUSTOM_PROTOBUF_CODECS.md) or [`cosmjs-types`](https://github.com/confio/cosmjs-types/blob/main/scripts/codegen.js) do is a good choice.
+You can choose which library you use to compile your Protobuf objects into TypeScript or JavaScript. Reproducing what [cosmjs-types](https://github.com/confio/cosmjs-types/blob/main/scripts/codegen.js) or [Stargate](https://github.com/cosmos/cosmjs/blob/main/packages/stargate/CUSTOM_PROTOBUF_CODECS.md) do is a good choice.
 
 ### Preparation
 
 This exercise assumes that:
 
-1. Your Protobuf definition files are in `./proto/myChain`.
-2. You want to compile them into TypeScript in `./client/src/types/generated`.
+1. The library you're creating is in 'myLib'.
+2. Your Protobuf definition files are in `./proto/`.
+3. You want to transpile them into TypeScript in `./src/codegen/`.
 
-Install `protoc` on your computer and its Typescript plugin in your project, possibly with the help of a Dockerfile:
-
-<CodeGroup>
-
-<CodeGroupItem title="Local">
+Install `telescope` on your computer globally:
 
 ```sh
-$ mkdir -p /usr/lib/protoc
-$ cd /usr/lib/protoc
-$ curl -L https://github.com/protocolbuffers/protobuf/releases/download/v21.7/protoc-21.7-linux-x86_64.zip -o protoc.zip
-$ unzip -o protoc.zip
-$ rm protoc.zip
-$ ln -s /usr/lib/protoc/bin/protoc /usr/local/bin/protoc
-$ npm install ts-proto@1.121.6 --save-dev
+npm install -g @cosmology/telescope
```

Comment:
We prefer to save locally in the project instead of `-g` globally. 
Is it a `--save-dev` here or not?
Please also set the `@x.y.z` version at which this update works. Also, I think we forgot a `--save-exact` here, if you could add it, so that we maximise reproducibility.
Also, keep the `$` as it is used everywhere in the tutorials I think.